stdlib/public/Distributed/DistributedActor.cpp

```diff
@@ -61,7 +61,10 @@ using TargetExecutorSignature =
                         /*witnessTables=*/void **,
                         /*numWitnessTables=*/size_t,
                         /*decoderType=*/Metadata *,
-                        /*decoderWitnessTable=*/void **),
+                        /*actorType=*/Metadata *,
+                        /*decoderWitnessTable=*/void **,
+                        /*distributedActorWitnessTable=*/void **
```

Comment:
This mismatch was a lot of "fun" (not) to debug.

Thank you @xedin for the help debugging here 🙏